tests/run-tests.spec.ts

```diff
@@ -458,76 +458,6 @@ test('should run all projects at once', async ({ activate }) => {
   ]);
 });
 
-test('should group projects by config', async ({ activate }) => {
```

Comment:
this test seems to be a remnant from old times. there's no grouping in the UI anymore, so I don't think it's valuable.